deployment/win-x86/package.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+source ../build.common.sh
+
+VERSION=`get_version ../..`
+
+package_portable ../.. `pwd`/dist/jellyfin_${VERSION}
+
+#TODO setup and maybe change above code to produce the Windows native zip format.
```

Comment:
`zip` can be done on Linux easily, and the existing ZIP files from @LogicalPhallacy just put all the files in there (though I'd prefer a subdir for consistency with most `zip`'d programs).